.github/update-sync-param-files.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import glob
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("output", type=str, help="output path of sync-param-files.yaml")
+    args = parser.parse_args()
+
+    # load current sync-param-files.yaml
+    with open(args.output) as f:
+        sync_param_file_curr = yaml.safe_load(f)
```

Comment:
[style] Please avoid uncommon abbreviations.

```suggestion
        sync_param_file_current = yaml.safe_load(f)
```

Reply:
Also, I personally feel this naming is better.

```suggestion
        current_sync_settings = yaml.safe_load(f)
```